src/Nethermind/Nethermind.Synchronization/FastSync/PendingSyncItems.cs

```diff
@@ -150,7 +150,7 @@ public void Clear()
 
         private bool TryTake(out StateSyncItem? node)
         {
-            for (int i = 0; i < _allStacks.Length; i++)
+            for (int i = 1; i < _allStacks.Length; i++)
```

Comment:
why does it start from 1?

Reply:
The first stack contains code requests which we handle separately.

Reply:
@tanishqjasoria does it make sense?

Reply:
yes, it make sense now.